README.md

```diff
@@ -45,18 +45,54 @@ With the IBM Cloud Operator, you can provision and bind [IBM public cloud servic
 <!-- SHOW operator hub -->
 
 ## Upgrading the operator
-To upgrade, you can reinstall the operator through the OperatorHub or the `curl` [installation command](README.md#setting-up-the-operator).
 
-### Upgrading to version 0.3.0 or later
+<!-- END SHOW operator hub -->
+
+Follow the instructions for the kind of installation you used: OperatorHub or the `curl` [installation command](README.md#setting-up-the-operator).
+
+### `curl` installation command
+
+To upgrade existing `curl` installs, you can reinstall the operator with the `curl` [installation command](README.md#setting-up-the-operator).
+If your current installation is version 0.1.x or 0.2.x, then follow these steps to complete a successful upgrade.
+
+1. [Copy your secrets and configmaps](#upgrading-to-version-v03-or-v10) to use the new names.
+2. Re-run the `curl` [installation command](#setting-up-the-operator) to install the latest version.
+3. Verify the new operator is running.
+    ```bash
+    kubectl get pods -n ibmcloud-operator-system
+    ```
+4. Verify `ibmcloud.ibm.com/v1` `Service` versions provision correctly.
```

Comment:
might have to put extra enters before and after the bash codeblock so that it renders properly in the operator hub view, at least I had to in other places